commit 10e0afa8fa25c9e0e8eaac9a0d730ab325086010
Author: Roberto Bagnara <bagnara@cs.unipr.it>
Date:   Sun Feb 12 08:48:59 2012 +0100

    Prolog sources must go under .../share/ppl as these files are architecture independent.
    Moreover, new versions of Automake no longer accept pkglib_DATA.

diff --git a/interfaces/Prolog/Ciao/Makefile.am b/interfaces/Prolog/Ciao/Makefile.am
index c8dd56e..8dcf17e 100644
--- a/interfaces/Prolog/Ciao/Makefile.am
+++ b/interfaces/Prolog/Ciao/Makefile.am
@@ -170,7 +170,7 @@ $(NO_UNDEFINED_FLAG) \
 -module \
 -avoid-version
 
-pkglib_DATA = ppl_ciao.po
+pkgdata_DATA = ppl_ciao.po
 
 ppl_ciao.pl: $(interface_generator_dependencies)
 	$(M4) --prefix-builtin -I../.. -I$(srcdir)/.. -I$(srcdir)/../.. \
diff --git a/interfaces/Prolog/GNU/Makefile.am b/interfaces/Prolog/GNU/Makefile.am
index 5904bf1..2ba45ee 100644
--- a/interfaces/Prolog/GNU/Makefile.am
+++ b/interfaces/Prolog/GNU/Makefile.am
@@ -169,7 +169,7 @@ ppl_gprolog.pl: $(interface_generator_dependencies)
 	$(M4) --prefix-builtin -I../.. -I$(srcdir)/.. -I$(srcdir)/../.. \
            $(srcdir)/ppl_interface_generator_gprolog_pl.m4 > $@
 
-pkglib_DATA = ppl_gprolog.pl
+pkgdata_DATA = ppl_gprolog.pl
 
 bin_PROGRAMS = ppl_gprolog
 
diff --git a/interfaces/Prolog/SICStus/Makefile.am b/interfaces/Prolog/SICStus/Makefile.am
index 17a54c7..22f809f 100644
--- a/interfaces/Prolog/SICStus/Makefile.am
+++ b/interfaces/Prolog/SICStus/Makefile.am
@@ -69,7 +69,7 @@ ppl_sicstus_SOURCES =
 
 if ENABLE_SHARED
 
-pkglib_DATA = ppl_sicstus.pl ppl_sicstus.s.o
+pkgdata_DATA = ppl_sicstus.pl ppl_sicstus.s.o
 
 ppl_sicstus.so: ppl_sicstus.pl $(top_builddir)/src/libppl.la $(WATCHDOG_LIBRARY) libppl_sicstus.la
 	$(LIBTOOL) --mode=link --tag=CXX \
@@ -87,7 +87,7 @@ install-data-local: ppl_sicstus.so
 
 else !ENABLE_SHARED
 
-pkglib_DATA = ppl_sicstus.pl
+pkgdata_DATA = ppl_sicstus.pl
 
 endif !ENABLE_SHARED
 
diff --git a/interfaces/Prolog/SWI/Makefile.am b/interfaces/Prolog/SWI/Makefile.am
index f5bcbe8..5e4e0e4 100644
--- a/interfaces/Prolog/SWI/Makefile.am
+++ b/interfaces/Prolog/SWI/Makefile.am
@@ -189,7 +189,7 @@ $(NO_UNDEFINED_FLAG) \
 bin_PROGRAMS = ppl_pl
 ppl_pl_SOURCES = ppl_pl.cc
 
-pkglib_DATA = ppl_swiprolog.pl
+pkgdata_DATA = ppl_swiprolog.pl
 
 CLPQ_TESTS_NONSTRICT = \
 ack.clpq \
diff --git a/interfaces/Prolog/XSB/Makefile.am b/interfaces/Prolog/XSB/Makefile.am
index 18648eb..1bc1e67 100644
--- a/interfaces/Prolog/XSB/Makefile.am
+++ b/interfaces/Prolog/XSB/Makefile.am
@@ -188,7 +188,7 @@ $(NO_UNDEFINED_FLAG) \
 -module \
 -avoid-version
 
-pkglib_DATA = ppl_xsb.xwam
+pkgdata_DATA = ppl_xsb.xwam
 
 ppl_xsb.xwam: ppl_xsb.H ppl_xsb.cc libppl_xsb.la
 	cp -f ppl_xsb.cc ppl_xsb.c
diff --git a/interfaces/Prolog/YAP/Makefile.am b/interfaces/Prolog/YAP/Makefile.am
index fab59c2..34d89e9 100644
--- a/interfaces/Prolog/YAP/Makefile.am
+++ b/interfaces/Prolog/YAP/Makefile.am
@@ -179,7 +179,7 @@ $(NO_UNDEFINED_FLAG) \
 -module \
 -avoid-version
 
-pkglib_DATA = ppl_yap.pl
+pkgdata_DATA = ppl_yap.pl
 
 CLPQ_TESTS_NONSTRICT = \
 ack.clpq \
